packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal.jsx

```diff
@@ -65,6 +65,8 @@ const FilterModal = () => {
         buy_sell_store.selected_payment_method_text ?? []
     );
     const [has_made_changes, setHasMadeChanges] = useSavedState('has_made_changes', false);
+    // eslint-disable-next-line no-unused-vars
+    const [is_matching, _] = useSavedState('has_made_changes', buy_sell_store.should_use_client_limits);
```

Comment:
didnt need to have a setter function here since we want to keep track of the original state of the observable without being affected by rerenders if we store it in a const